### PR TITLE
Add v3.0 observability roadmap

### PR DESCRIPTION
**Pull request by benjaminhuo**
Signed-off-by: Benjamin <benjamin@yunify.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add v3.0 observability roadmap

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -32,6 +33,14 @@ KubeSphere Roadmap demonstrates a list of open source product development plans
 
 #### Observability
 
+- [ ] Logging console enhancement
+- [ ] Monitoring stack upgrade including Prometheus, Prometheus Operator, Node exporter, kube-state-metrics etc.
+- [ ] Custom metrics support including application custom metrics dashboard, custom metrics HPA
+- [ ] Integration with Alertmanager
+- [ ] K8s Event management
+- [ ] K8s Audit Support
+- [ ] Notification Enhancement
```
Comment (FeynmanZhou) [inline docs/roadmap.md]: "Notification Enhancement" is too vague, could you add more detailed description?  e.g. add new notification channels including Wechat...

File: docs/roadmap.md [hunk 2]
```diff
@@ -32,6 +33,14 @@ KubeSphere Roadmap demonstrates a list of open source product development plans
 
 #### Observability
 
+- [ ] Logging console enhancement
+- [ ] Monitoring stack upgrade including Prometheus, Prometheus Operator, Node exporter, kube-state-metrics etc.
+- [ ] Custom metrics support including application custom metrics dashboard, custom metrics HPA
+- [ ] Integration with Alertmanager
+- [ ] K8s Event management
+- [ ] K8s Audit Support
```
Comment (FeynmanZhou) [inline docs/roadmap.md]: Any more information or reference regarding K8s Event and Audit? Do you plan to integrate native upstream k8s event and audit feature, will any new KS APIs to add? @benjaminhuo

File: docs/roadmap.md [hunk 3]
```diff
@@ -32,6 +33,14 @@ KubeSphere Roadmap demonstrates a list of open source product development plans
 
 #### Observability
 
+- [ ] Logging console enhancement
+- [ ] Monitoring stack upgrade including Prometheus, Prometheus Operator, Node exporter, kube-state-metrics etc.
+- [ ] Custom metrics support including application custom metrics dashboard, custom metrics HPA
+- [ ] Integration with Alertmanager
```
Comment (FeynmanZhou) [inline docs/roadmap.md]: Any more information or reference regarding Alertmanager? Do we have enhancement for this integration? @benjaminhuo

File: docs/roadmap.md [hunk 4]
```diff
@@ -32,6 +33,14 @@ KubeSphere Roadmap demonstrates a list of open source product development plans
 
 #### Observability
 
+- [ ] Logging console enhancement
```
Comment (FeynmanZhou) [inline docs/roadmap.md]: Logging console is a big module, which aspect or point will be enhance within logging console? @benjaminhuo

**Comment by benjaminhuo** [comment]
I think we needn't to elaborate on those items because it's a rough plan and might be adjusted slightly. @FeynmanZhou

**Pull request merged**